.github/workflows/build-and-deploy.yml

```diff
@@ -286,6 +286,23 @@ jobs:
           deployment_id: ${{ steps.deployment.outputs.deployment_id }}
           env_url: ${{ steps.deploy_v2_review.outputs.deploy-url }}
 
+  merge-dependabot:
+    name: Merge dependabot
+    if: ${{ github.event_name == 'pull_request' && contains(github.event.pull_request.labels.*.name, 'dependencies') }}
+    needs: [test, deploy-v2-review-app]
```

Comment:
probably should point too deploy_review until we start migrating environments to aks

will also need an addition to the deploy_review if statement, so that it runs on dependencies
i.e.  || contains(github.event.pull_request.labels.*.name, 'dependencies'))